exchanges/stream/buffer/buffer_types.go

```diff
@@ -19,10 +39,17 @@ type Orderbook struct {
 	sortBufferByUpdateIDs bool // When timestamps aren't provided, an id can help sort
 	updateEntriesByID     bool // Use the update IDs to match ob entries
 	exchangeName          string
-	dataHandler           chan interface{}
+	dataHandler           chan<- interface{}
```

Comment:
Excuse my ignorance, but what's happening here?

Reply:
It restricts the channel to only allow data to be put into it. As the reading from the channel in this package is unnecessary. You can also restrict it another way by going `<-chan interface{}` this only allows you to read from it only.

Reply:
Thanks! Been awhile since I've come across that :D